### PR TITLE
Taras maksymiv

### PR DESCRIPTION
**Pull request by TarasMaksymiv**

### CODE REVIEW

File: tmaks/HW5/hw5_2.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (kolyasalubov) [inline tmaks/HW5/hw5_2.py:1]: good job

File: tmaks/HW6/hw6_1.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+even_list = []
```
Comment (kolyasalubov) [inline tmaks/HW6/hw6_1.py:2]: another way:

even = []
odd = []
other = []

for number in range(1, 11):
    if number >= 2 and number % 2 == 0:
        even.append(number)
    elif number >= 3 and number % 3 == 0:
        odd.append(number)
    else:
        other.append(number)

print(f"Even numbers that are divisible by 2: {even}")
print(f"\nOdd numbers, which are divisible by 3: {odd}")
print(f"\nNumbers that are not divisible by 2 and 3: {other}")

File: tmaks/HW6/hw6_2.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (kolyasalubov) [inline tmaks/HW6/hw6_2.py:1]: good job

File: tmaks/HW7/PracticalTask/12_Is_this_my_tail.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (kolyasalubov) [inline tmaks/HW7/PracticalTask/12_Is_this_my_tail.py:1]: good job

File: tmaks/HW7/PracticalTask/7_Multiples_of_3_and_5.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (kolyasalubov) [inline tmaks/HW7/PracticalTask/7_Multiples_of_3_and_5.py:1]: good job

File: tmaks/HW7/PracticalTask/8_Will_you_make_it.py [hunk 6]
```diff
@@ -0,0 +1,5 @@
+def zero_fuel(distance_to_pump, mpg, fuel_left):
```
Comment (kolyasalubov) [inline tmaks/HW7/PracticalTask/8_Will_you_make_it.py:1]: good job

File: tmaks/HW7/PracticalTask/9_Are_You_Playing_Banjo.py [hunk 7]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (kolyasalubov) [inline tmaks/HW7/PracticalTask/9_Are_You_Playing_Banjo.py:1]: good job

File: tmaks/HW7/PracticalTask/Convert_a_Number_to_a_String.py [hunk 8]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (kolyasalubov) [inline tmaks/HW7/PracticalTask/Convert_a_Number_to_a_String.py:1]: good job

File: tmaks/HW7/PracticalTask/Find_The_Distance_Between_Two_Points.py [hunk 9]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (kolyasalubov) [inline tmaks/HW7/PracticalTask/Find_The_Distance_Between_Two_Points.py:1]: good job

File: tmaks/HW7/PracticalTask/Jenny's_secret_message.py [hunk 10]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (kolyasalubov) [inline tmaks/HW7/PracticalTask/Jenny's_secret_message.py:1]: good job

File: tmaks/HW7/PracticalTask/No_yelling.py [hunk 11]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (kolyasalubov) [inline tmaks/HW7/PracticalTask/No_yelling.py:1]: good job

File: tmaks/HW7/PracticalTask/Reverse_List_Order.py [hunk 12]
```diff
@@ -0,0 +1,2 @@
+def reverse_list(l):
```
Comment (kolyasalubov) [inline tmaks/HW7/PracticalTask/Reverse_List_Order.py:1]: good job

File: tmaks/HW7/PracticalTask/Reversing_Words_in_a_String.py [hunk 13]
```diff
@@ -0,0 +1,16 @@
+st = "d ioje d yrdsfykkhwuoookjhesgdj atyetqwklslgtalediqrlrurldrtlfsjfey"
+
+
```
Comment (kolyasalubov) [inline tmaks/HW7/PracticalTask/Reversing_Words_in_a_String.py:3]: good job

File: tmaks/HW7/hw7_1.py [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+#
+#
+#first_number = int(input("Enter firs number: "))
+#second_number = int(input("Enter second number: "))
+#
+#print(largest_numberV1(first_number, second_number))
+#print(largest_numberV2(first_number, second_number))
+
+#------------- Without using input -------
+
```
Comment (kolyasalubov) [inline tmaks/HW7/hw7_1.py:33]: good job

File: tmaks/HW7/hw7_2.py [hunk 15]
```diff
@@ -0,0 +1,35 @@
+
+
```
Comment (kolyasalubov) [inline tmaks/HW7/hw7_2.py:2]: good job

File: tmaks/HW7/hw7_3.py [hunk 16]
```diff
@@ -0,0 +1,20 @@
+
+
```
Comment (kolyasalubov) [inline tmaks/HW7/hw7_3.py:2]: excellent work

File: tmaks/HW7/PracticalTask/10_Convert_boolean_values_to_strings_'Yes'_or_'No'.py [hunk 17]
```diff
@@ -0,0 +1,2 @@
+def bool_to_word(boolean):
```
[the remark this answers is not in the document]
Comment (TarasMaksymiv) [reply tmaks/HW7/PracticalTask/10_Convert_boolean_values_to_strings_'Yes'_or_'No'.py:1]: +

File: tmaks/HW7/PracticalTask/11_Counting_sheep....py [hunk 18]
```diff
@@ -0,0 +1,7 @@
+
```
[the remark this answers is not in the document]
Comment (TarasMaksymiv) [reply tmaks/HW7/PracticalTask/11_Counting_sheep....py:1]: +

File: tmaks/HW7/PracticalTask/12_Is_this_my_tail.py (hunk 4 above)
Comment (TarasMaksymiv) [reply tmaks/HW7/PracticalTask/12_Is_this_my_tail.py:1]: +

File: tmaks/HW7/PracticalTask/7_Multiples_of_3_and_5.py (hunk 5 above)
Comment (TarasMaksymiv) [reply tmaks/HW7/PracticalTask/7_Multiples_of_3_and_5.py:1]: +

File: tmaks/HW7/PracticalTask/8_Will_you_make_it.py (hunk 6 above)
Comment (TarasMaksymiv) [reply tmaks/HW7/PracticalTask/8_Will_you_make_it.py:1]: +

File: tmaks/HW7/PracticalTask/9_Are_You_Playing_Banjo.py (hunk 7 above)
Comment (TarasMaksymiv) [reply tmaks/HW7/PracticalTask/9_Are_You_Playing_Banjo.py:1]: +

File: tmaks/HW7/PracticalTask/Convert_a_Number_to_a_String.py (hunk 8 above)
Comment (TarasMaksymiv) [reply tmaks/HW7/PracticalTask/Convert_a_Number_to_a_String.py:1]: +

File: tmaks/HW7/PracticalTask/Find_The_Distance_Between_Two_Points.py (hunk 9 above)
Comment (TarasMaksymiv) [reply tmaks/HW7/PracticalTask/Find_The_Distance_Between_Two_Points.py:1]: +

File: tmaks/HW7/PracticalTask/Jenny's_secret_message.py (hunk 10 above)
Comment (TarasMaksymiv) [reply tmaks/HW7/PracticalTask/Jenny's_secret_message.py:1]: +

File: tmaks/HW7/PracticalTask/No_yelling.py (hunk 11 above)
Comment (TarasMaksymiv) [reply tmaks/HW7/PracticalTask/No_yelling.py:1]: +

File: tmaks/HW7/PracticalTask/Reverse_List_Order.py (hunk 12 above)
Comment (TarasMaksymiv) [reply tmaks/HW7/PracticalTask/Reverse_List_Order.py:1]: +

File: tmaks/HW7/PracticalTask/Reversing_Words_in_a_String.py (hunk 13 above)
Comment (TarasMaksymiv) [reply tmaks/HW7/PracticalTask/Reversing_Words_in_a_String.py:3]: +

File: tmaks/HW7/hw7_1.py (hunk 14 above)
Comment (TarasMaksymiv) [reply tmaks/HW7/hw7_1.py:33]: +

File: tmaks/HW7/hw7_2.py (hunk 15 above)
Comment (TarasMaksymiv) [reply tmaks/HW7/hw7_2.py:2]: +

File: tmaks/HW7/hw7_3.py (hunk 16 above)
Comment (TarasMaksymiv) [reply tmaks/HW7/hw7_3.py:2]: +

**Pull request merged**